package.json

```diff
@@ -43,7 +43,7 @@
 		"define-properties": "^1.1.2",
 		"function-bind": "^1.1.1",
 		"has-symbols": "^1.0.0",
-		"object-keys": "^1.0.11"
```

Comment:
Note that the `^` makes this unnecessary, as everything will automatically update it to the latest v1 (but not v2).

Reply:
Technically yes. But, in order to resolve issues with the the latest version of chrome this package needs to have version `1.0.12` or greater.